llvm/lib/Analysis/IVDescriptors.cpp

```diff
@@ -784,17 +784,17 @@ RecurrenceDescriptor::isMinMaxPattern(Instruction *I, RecurKind Kind,
     return InstDesc(Kind == RecurKind::FMin, I);
   if (match(I, m_OrdOrUnordFMax(m_Value(), m_Value())))
     return InstDesc(Kind == RecurKind::FMax, I);
-  if (match(I, m_Intrinsic<Intrinsic::minnum>(m_Value(), m_Value())))
+  if (match(I, m_FMin(m_Value(), m_Value())))
     return InstDesc(Kind == RecurKind::FMin, I);
-  if (match(I, m_Intrinsic<Intrinsic::maxnum>(m_Value(), m_Value())))
+  if (match(I, m_FMax(m_Value(), m_Value())))
```

Comment:
Can we please rename these to F_Maxnum? Now that we have imported the full min-max zoo, we should be more explicit about this...

Reply:
Sounds good, I think I'll do this in a separate PR

Reply:
Ok turns out the matchers weren't used before this PR, so I pulled in the update here